### DISCUSSION TITLE
Lifecycle for "well-known URI" updates

### DISCUSSION THREAD

**Discussion by matatk** [w3c/personalization-semantics#251] [Ideas]
What will the process be for making updates to the use of destinations (#250) in well-known URIs?

Note: we have some prior art to refer to here: https://www.w3.org/TR/change-password-url/

Thanks for the question, @abhina-kumar

**Comment by abhina-kumar** [comment]
> What will the process be for making updates to the use of destinations (#250) in well-known URIs?
> 
> Note: we have some prior art to refer to here: https://www.w3.org/TR/change-password-url/
> 
> Thanks for the question, @abhina-kumar

@matatk - IMO the use case here is different than spec for well-known url for `change-password`.  `change-password` deals specifically with concrete use case.

In this case, there can be number of well-known urls like one for `log-in`, other for `log-out` etc. The question is  - Will all these part of single IEFT RFC for well-known-url or multiple? And how a new url will get added (eg. what happens when the community wants to add another well known url for `search` capability)?

**Reply by matatk** [reply]
The idea with our spec is that we'll start with a defined list of destinations, which come from the COGA TF, and make sense as very popular/general waypoints (e.g. 'contact', 'accessibility statement', 'log in', ...)

Over time, we may extend that list in our spec.

At _any_ time, anyone can register a well-known URL with IANA, so the community can do that without us specifying them in W3C space. We would tend to just specify the ones that we know have consensus (or even more) support across most sites.

This sort of approach (specifying the 'core', and letting the community decide on the rest) has worked in areas such as EPUB accessibility, where they have a core schema for accessibility metadata, and the community has extended it. Occasionally, things are brought into the (specified) core, as it becomes clear that they have wide community consensus. I think this spec may work similarly - though we should keep focused on having a genuinely small (and therefore more helpful to the user) set of destinations.

**Reply by abhina-kumar** [reply]
@matatk: But we also want them to be part of `/.well-known/ia/` (to tell the user _which_ URLs are supported)....Isn't it?
